### ISSUE TITLE
tabs on wallet/swap page are not real tabs

### ISSUE THREAD

**Issue by rolznz** [getAlby/hub#1670]
We have a shadcn tab component now. That will also fix accessibility (they cannot be accessed via tabbing on the keyboard)
